app/controllers/questions_controller.rb

```diff
@@ -1,4 +1,7 @@
 class QuestionsController < ApplicationController
+    def set_current_user
+        User.current = current_user
+      end
```

Comment:
Small comment, but proper alignment of this methods contents and `end` and a line break after this method will help keep your code clear.